.env.default

```diff
@@ -3,6 +3,7 @@ AUTH0_CLIENT_ID=
 AUTH0_CLIENT_SECRET=
 AUTH0_ACCESS_TOKEN_ENDPOINT=
 AUTH0_AUDIENCE=
+CONNECTION_NAME=
```

Comment:
the variable will need to be added to harness

Reply:
We could leave here a default value AND set it in harness. The harness one will override the default that is set here, since dotenv gives higher precedence to env vars set at the OS level.